packages/cli-platform-ios/src/commands/runIOS/index.ts

```diff
@@ -230,7 +234,10 @@ async function runOnSimulator(
           findMatchingSimulator(simulators, {simulator: fallback})
         );
       },
-      findMatchingSimulator(simulators, args),
+      findMatchingSimulator(simulators, {
+        ...args,
```

Comment:
The only other key in this object argument is `uuid`, we can be explicit instead of spreading `args` in this case I guess